tests/test_crud/crud_test.py

```diff
@@ -0,0 +1,103 @@
+import unittest
+import unittest.mock
+
+from fastapi.testclient import TestClient
+
+from alws.app import app
+from alws.dependencies import get_db
+from alws.utils import jwt_utils
+from alws.config import settings
+
+
+class TestPingTasks(unittest.TestCase):
+
+    client = TestClient(app)
+    token = jwt_utils.generate_JWT_token({'user_id': 1}, settings.jwt_secret, 'HS256')
+
+    def test_empy_tasks(self):
+        
+        response = self.client.post(
+            "/api/v1/build_node/ping",
+            headers={"Authorization": f"Bearer {self.token}"},
+            json={'active_tasks': []}
+        )
+        message = "Empty active_tasks aren't pinged"
+        self.assertEqual(response.status_code, 200, message)
+
+
+    async def custom_db(self):
+        class MockedDB(unittest.mock.AsyncMock):
+            async def __aenter__(self):
+                return self
+
+            async def __aexit__(self, exc_type, exc_val, exc_tb):
+                pass
+
+            def begin(self):
+                return self
+
+        return MockedDB()
+
+
+    def test_db_call(self):
+        app.dependency_overrides[get_db] = self.custom_db
+        with open('test_logs', 'a') as logs:
+            logs.write('self.custom_db: {}\n'.format(self.custom_db))
+        response = self.client.post(
+            "/api/v1/build_node/ping",
+            headers={"Authorization": f"Bearer {self.token}"},
+            json={'active_tasks': [1, 2, 3]}
+        )
+        message = "Tasks aren't pinged"
+        self.assertEqual(response.status_code, 200, message)
+
+
+class TestCreatePlatform(unittest.TestCase):
+
+    client = TestClient(app)
+    token = jwt_utils.generate_JWT_token({'user_id': 1}, settings.jwt_secret, 'HS256')
+
+    async def custom_db(self):
```

Comment:
Create a base class with this method derived from TestCase, then inherit it for other tests that need database.